### ISSUE TITLE
Version 0.1.4 can not find the keys after the update

### ISSUE THREAD

**Issue by AlexeyALeonov** [GooBox/goobox-community-gui#113]
### Versions

Replace the values below with your own:

- **GUI Version:** v0.1.4
- **Operating System:** windows 10
- **Storage provider selected:** Storj

Optional:

- **Current account balance:** <number of units>
- **Current wallet address:** <address>
- **Size of sync directory:** 78.5MB
- **Number of files and folders in Sync directory:** 6 files and 4 folders

### logs

Please attach the following logs:

## windows

- `C:\Users\USER\AppData\Roaming\Goobox\<log>.log`
- `C:\Users\USER\AppData\Local\Goobox\Logs\<...>-sync.log`
```
[2018-03-10 17:27:04.421] [verbose] [main] INFO  io.goobox.sync.storj.App - No keys found. Waiting for keys to be imported.
[2018-03-10 17:27:07.423] [verbose] [main] INFO  io.goobox.sync.storj.App - No keys found. Waiting for keys to be imported.
```

### Expected Behavior

Please describe the program's expected behavior.

```
Working synchronization after the update.
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
Infinity cycle of synchronization without success.
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Have installed and working 0.1.3.
2. Upgrade to 0.1.4
3. Infinity sync cycle because of
```
[2018-03-10 17:27:04.421] [verbose] [main] INFO  io.goobox.sync.storj.App - No keys found. Waiting for keys to be imported.
[2018-03-10 17:27:07.423] [verbose] [main] INFO  io.goobox.sync.storj.App - No keys found. Waiting for keys to be imported.
```

**Comment by MeijeSibbel** [comment]
Thanks!, related to https://github.com/GooBox/goobox-community-gui/issues/112

**Comment by jkawamoto** [comment]
This issue has been fixed in v0.1.5.

If the log still has `No keys found. Waiting for keys to be imported.`, please delete `C:\Users\USER\AppData\Roaming\Goobox\storage\config.json` and restart the app.
